packages/account/src/Components/currency-selector/currency-selector.tsx

```diff
@@ -92,8 +93,8 @@ const CurrencySelector = observer(
 
         const has_currency = Boolean(currency);
 
-        const { real_account_signup, real_account_signup_target, resetRealAccountSignupParams, is_desktop, is_mobile } =
-            ui;
+        const { real_account_signup, real_account_signup_target, resetRealAccountSignupParams } = ui;
+        const { isMobile, isDesktop } = useDevice();
 
```

Comment:
@fasihali-deriv THis component is part of real-account signup flow. Please check with Sergie if he is already working on this

Reply:
Oh alright. 
I thought this is related to POI. 
Thanks for highlighting